engine/items/item_effects.asm

```diff
@@ -192,6 +193,7 @@ ItemEffects:
 	dw PokeBallEffect      ; PARK_BALL
 	dw NoEffect            ; RAINBOW_WING
 	dw NoEffect            ; ITEM_B3
+	check_table_length ITEM_B3
```

Comment:
Why `ITEM_B3`? Is there any reason to stop here? Is there a more suitable constant for it?

Reply:
The items past `ITEM_B3`, from `BRICK_PIECE` to `ITEM_BE` (mostly Mails, plus the TMs and HMs), don't have effect pointers. ¯\\\_(ツ)_/¯ It doesn't cause problems because they're all `ITEMMENU_NOUSE`. I'll add a comment about that.